### PR TITLE
Add a tutorial on creating a simple Grafana dashboard

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Added a dashboard on creating and configuring a simple Grafana dashboard of a Gauge type.

**Related issue(s)**
See also #4884

### CODE REVIEW

File: docs/monitoring/08-02-create-and-configure-grafana-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+3. Select **Add Query**.
+
+![Add Query](./assets/add-query.png)
+
+4. Select Prometheus data source from the **Queries to** drop-down list and choose the `cpu_temperature_celsius` metric.
+
+![New dashboard](./assets/new-dashboard.png)
+
+5. Select the **Instant** query to be able to retrieve the last value for each time series.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/monitoring/08-02-create-and-configure-grafana-dashboard.md]: You are right, that's not too clear :) I will update it. The idea is that this option allows you to retrieve the "instant" data from the metric.

File: docs/monitoring/08-02-create-and-configure-grafana-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+This tutorial shows how to create and configure a basic Grafana dashboard of a [Gauge](https://grafana.com/docs/features/panels/singlestat/#gauge) type. The dashboard shows how the values of the `cpu_temperature_celsius` metric change in time, representing the current processor temperature ranging from `60` to `90` Celsius degrees. The dashboard shows explicitly when the CPU temperature exceeds the pre-defined threshold of `75` Celsius degrees.
+
+## Prerequisites
+
+This tutorial is a follow-up of the [**Observe application metrics**](#tutorials-observe-application-metrics) tutorial that uses the `monitoring-custom-metrics` example. This example deploys the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
+
+## Steps
+
+Access the Grafana UI, select the `cpu_temperature_celsius` metric, the Gauge dashboard type, and start the configuration.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/monitoring/08-02-create-and-configure-grafana-dashboard.md]: Hmm, I will think about improving that.

File: docs/monitoring/08-02-create-and-configure-grafana-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+![Gauge dashboard type](./assets/gauge-dashboard-type.png)
+
+7. Save the changes by clicking the disk icon in the top right corner of the page. Provide a name for the dashboard.
+
+![Save the dashboard](./assets/save-dashboard.png)
+
+### Configure the dashboard
+
+To edit the dashboard settings, click the **Panel Title** and select **Edit**.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/monitoring/08-02-create-and-configure-grafana-dashboard.md]: I choose the second option :)

File: docs/monitoring/08-02-create-and-configure-grafana-dashboard.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/monitoring/08-02-create-and-configure-grafana-dashboard.md]: `Follow these sections to create the Gauge dashboard type for the `cpu_temperature_celsius` metric and check if it works.`

File: docs/monitoring/08-02-create-and-configure-grafana-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+2. Click the **+** icon on the left sidebar and select **Dashboard** from the **Create** menu.
+
+![Create a dashboard](./assets/create-dashboard.png)
+
+3. Select **Add Query**.
+
+![Add Query](./assets/add-query.png)
+
+4. Select Prometheus data source from the **Queries to** drop-down list and choose the `cpu_temperature_celsius` metric.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/monitoring/08-02-create-and-configure-grafana-dashboard.md]: I left select but I changed choose into "pick".

File: docs/monitoring/08-02-create-and-configure-grafana-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+This tutorial is a follow-up of the [**Observe application metrics**](#tutorials-observe-application-metrics) tutorial that uses the `monitoring-custom-metrics` example. This example deploys the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
+
+## Steps
+
+Follow these sections to create the Gauge dashboard type for the `cpu_temperature_celsius` metric.
+
+### Create the dashboard
+
+1. Navigate to Grafana. It is available under the `https://grafana.{DOMAIN}` address, where `{DOMAIN}` is the domain of your Kyma cluster, such as `https://grafana.34.63.57.190.xip.io` or `https://grafana.nightly.a.build.kyma-project.io/`. To access it from the Console UI, click **Stats & Metrics** on the left navigation menu.
```
Comment (michal-hudy) [inline docs/monitoring/08-02-create-and-configure-grafana-dashboard.md]: I'm not sure if we should mention nightly cluster in examples as it is not available for everyone.

Comment (kazydek) [reply docs/monitoring/08-02-create-and-configure-grafana-dashboard.md]: As discussed, I ill change it into `http://kyma.example.com/`.

**Pull request merged**